e2e/cypress/support/commands/form.js

```diff
@@ -27,8 +27,9 @@ Cypress.Commands.add(
       .waitForRoute(apiRoute)
       .then((xhr) => {
         const recordId = xhr.response.body.pid_value;
+        const controlNumber = xhr.response.body.control_number;
 
-        if (recordId) {
+        if (recordId || controlNumber) {
```

Comment:
why we need to add controlNumber? can we reuse the `recordId` as everywhere else?

Reply:
I changed the implementation a bit since this comment, but basically I use controlNumber here instead of recordId in order to differentiate submissions that result in opening record editor from the rest (as only they have control_number field: institutions, experiments, journals). I need to separate them because they are tested differently than the other submissions

Reply:
Thanks! Do you mean here https://github.com/inspirehep/inspirehep/pull/2601/files#diff-0c95d7b3fb6f89e886899bd38a35b1a535e1eb1cdb06c48e78cfdf29806b50cdR32?

Reply:
Yes, this file, in this condition below: https://github.com/inspirehep/inspirehep/pull/2601/files#diff-0c95d7b3fb6f89e886899bd38a35b1a535e1eb1cdb06c48e78cfdf29806b50cdR37

Reply:
IMO we should just check and use recordId for both cases, it's confusing to have different naming for the same thing (i.e. the record id)

Reply:
you're right, I will change that